### DISCUSSION TITLE
Is it possible to make parallel at the same y in sequence diagram?

### DISCUSSION THREAD

**Discussion by maple3142** [mermaid-js/mermaid#5528] [Q&A]
For example, on https://sequencediagram.org/

```
Alice->Bob: Message 1
parallel
note over Alice: note 1
note over Bob: note 2
parallel off
parallel 
Bob->Alice: hello 1
Bob->Eve: hello 2
parallel off
```

shows

![image](https://github.com/mermaid-js/mermaid/assets/9370547/a55e4f9e-4061-4434-b769-801e68c1c552)

And the equivalent in mermaid would be:

```
sequenceDiagram
    Alice->>Bob: Message 1
    par
    note over Alice: note 1
    note over Bob: note 2
    end
    par
    Bob->>Alice: hello 1
    Bob->>Eve: hello 2
    end
```

![image](https://github.com/mermaid-js/mermaid/assets/9370547/a8e60537-042e-4ca0-a7ce-a86018d33ec4)

I would like to make those two notes and two arrows actually shows on the same y instead of how currently mermaid does that. Is it possible in mermaid?

**Comment by manoelpqueiroz** [comment]
I'm also in the same boat. The behaviour from sequencediagram.org (going as far as not showing the `par` box) would be perfect for enhancing the original purpose of the diagram.

In my use case, for example, I am attempting to use the sequence diagram to list all members of a team, using notes to describe their activities and arrows to describe activities that are being done by someone but should actually be done by another person. Making notes work actually parallel on the y axis would reduce the size of the final diagram.
